beacon-chain/blockchain/testing/mock.go

```diff
@@ -182,7 +183,7 @@ func (s *ChainService) ReceiveBlockInitialSync(ctx context.Context, block interf
 	}
 	parentRoot := block.Block().ParentRoot()
 	if !bytes.Equal(s.Root, parentRoot[:]) {
-		return errors.Errorf("wanted %#x but got %#x", s.Root, block.Block().ParentRoot())
+		return errors.Errorf("ReceiveBlockInitialSync wanted %#x but got %#x", s.Root, block.Block().ParentRoot())
```

Comment:
is this an intended change or just for debug?

Reply:
I think we had a few "wanted x but got y" error messages that made it difficult to suss out the source of the error. I can see not liking the func name in the error message but it's nice to have errors that can be traced back to their source. I don't feel strongly on whether to revert this change or leave it in.

Reply:
I don't know what is idiomatic here, but I haven't seen much of the copying/pasting function as it is. I thought the common pattern would be more descriptive `failed to compare root in receive block initial sync`... etc but I don't feel strongly either

Reply:
reverting this